### ISSUE TITLE
SSL CONNECTION ERROR 

### ISSUE THREAD

**Issue by androwg** [rainforestapp/tester-chrome-extension#44]
Terminal gives SSL CONNECTION ERROR when try browsing any site

**Comment by jbarber** [comment]
Can you provide more information?

**Comment by androwg** [comment]
@jbarber when i copied the url to terminal it says SSL CONNECTION ERROR i reported the problem in hit 765266224 also this hit appears strange in my history !
![untitled](https://cloud.githubusercontent.com/assets/15823279/12508847/1d6c31b8-c107-11e5-842d-473dfb9b0939.png)

**Comment by androwg** [comment]
got a good working hit now without this problem i want to note in the previous hit 765266224 which i had the problem in , the terminal for the first time i see was very large taking almost the whole page i don't know if this has a relation to the problem SSL CONNECTION ERROR  or not

**Comment by JPierreS** [comment]
I got that issue yesterday as well, for several consecutive hits, but since then things went back to normal.

The SSL issue is not something new, it used to happen before over the months though very rarely.  But yesterday the issue was more persistent for a while.

I got some RF API hits yesterday that worked, but the SSL errors occured on mostly non RF API hits.

Today friday 22nd as far as I'm concerned, I have not seen the SSL error yet :)

In fact I haven't seen any error so far today, so I must be in a dream then :-))) someone wake me up!

:)
